app/components/concept_details_component.rb

```diff
@@ -69,11 +69,12 @@ def properties_set_by_keys(keys, concept_properties, exclude_keys = [])
     end
   end
 
-  def filter_properties(top_keys, bottom_keys, exclude_keys, concept_properties)
+  def filter_properties(top_keys, bottom_keys, exclude_keys, concept_properties) 
     all_keys = concept_properties&.keys || []
     top_set = properties_set_by_keys(top_keys, concept_properties, exclude_keys)
     bottom_set = properties_set_by_keys(bottom_keys, concept_properties, exclude_keys)
     leftover = properties_set_by_keys(all_keys - top_keys - bottom_keys, concept_properties, exclude_keys)
```

Comment:
here the `leftover`, had to be `all_keys - top_keys - bottom_keys`, so it should not include at all `top_set` keys zt first no need to reject after, so the issue is in this line.

Reply:
these '-' operations are not working because all_keys has prefixes:
`all_keys = ["dct:description",
 "dcat:keyword",
 "dcat:landingPage",
 "dct:type",
 "dct:title",
 "owl:versionInfo",
 "dct:language",
 "dct:issued",
 "skos:hasTopConcept",
 "dct:modified",
 "adms:versionNotes",
 "owl:versionIRI"]`
 
 but top_keys, and bottom_keys do not:
 `top_keys=["description", "comment"]`

Reply:
Ok, so when we added the prefixes (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/393) we broke this.

To complete your solution, you will need to reject also  `bottom_keys` so that `leftover` represents the original goal.